### PR TITLE
optimize mdw usage

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: logic/mdwLogic.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
+const logger = require('../utils/logger')(module);
+
+const LIMIT = 100; // max 1000
+const knownHashes = {};
+
+module.exports = class MdwLogic {
+  // fetches pages forwards, if no next its the last page, don't cache that
+  static async iterateMdw(next, abortIfHashKnown = false) {
```
Comment (kenodressel) [inline logic/mdwLogic.js:9]: I would argue we should default `abortIfHashKnown` to `true`

File: logic/mdwLogic.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    const url = `${process.env.MIDDLEWARE_URL}/${next}`;
```
Comment (kenodressel) [inline logic/mdwLogic.js:10]: no default url?

**Review by kenodressel — APPROVED** [review]

File: logic/mdwLogic.js (hunk 1 above)
Comment (thepiwo) [reply logic/mdwLogic.js:9]: no, as this is only used for backwards fetch of events, iterateMdw is also used for names

File: logic/mdwLogic.js (hunk 2 above)
Comment (thepiwo) [reply logic/mdwLogic.js:10]: not needed probably, what do you think?

Comment (kenodressel) [reply logic/mdwLogic.js:10]: Lets leave it as is and at some point we can move everything to a proper config file with defaults.

**Pull request merged**